erc777.k

```diff
@@ -9,58 +9,61 @@ module ERC777
   imports MAP
   imports SET
   imports LIST
+  imports KCELLS
 
   configuration
      <ERC777>
+       <caller> 0:Address </caller>
        <k> $PGM:K </k>
-       <balances> .Map </balances> // holder -> balance
-       <operators> .Map </operators> // holder -> 2^operator
-       <senderHooks> .Map </senderHooks> // holder -> senderHook
-       <receiverHooks> .Map </receiverHooks> // holder -> receiverHook
+       <accounts>
+         <account multiplicity="*">
+           <id> 0:Address </id>
+           <balance> 0:Number </balance>
+           <isContract> .K </isContract>
+           <hookFail> false </hookFail>
+
+           //ERC777 operators
+           //fixme something is missing here - how a set can hold "holder -> 2^operator"?
+           <operators> .Set </operators> // holder -> 2^operator
```

Comment:
to be consistent, i'd like to have `<operators>` in the similar way with `<allowances>`.